spec/book_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../book'
+
+describe Book do
+  before :each do
+    @book = Book.new('publisher', 'cover_state', '2022/2/3')
+  end
```

Comment:
[OPTIONAL] You could add instance variables for the publisher and cover_state strings (@ publisher = 'publisher') and add those instance variables where you have hard coded the strings. That way if you ever need to change the stings you don't have to hard code your test again.